depinject/container.go

```diff
@@ -95,8 +95,6 @@ func (c *container) call(provider *ProviderDescriptor, moduleKey *moduleKey) ([]
 }
 
 func (c *container) getResolver(typ reflect.Type, key *moduleKey) (resolver, error) {
-	c.logf("Resolving %v", typ)
```

Comment:
I accidentally merged this debug code at some point, it makes error messages needlessly verbose and log.  Sorry about that.